### DISCUSSION TITLE
How to delete image from rich editor content.

### DISCUSSION THREAD

**Discussion by bumpyy** [filamentphp/filament#6721] [Help] [answered]
This is self-explanatory, but I can't find a guide to delete image from rich editor, the images remain in the storage folder if the image edited, deleted, or even after the "Post" got deleted.

**Comment by danharrin** [comment, marked as answer]
Filament doesn't know where else you might be using this image, so we don't touch it. If you really need this, you should use a model observer to track changes to the rich editor content and delete the corresponding files. But personally I don't think it's worth it.

**Reply by bumpyy** [reply]
yeah..., I think you're right.
